flow/genericactors.actor.h

```diff
@@ -300,6 +300,14 @@ Future<Void> store(T& out, Future<T> what) {
 	});
 }
 
+template <class A, class... Bs>
```

Comment:
Is `storeTuple` still needed?

Reply:
I'm thinking about this. I agree that creating a structure for a complex return value is helpful for understanding, but sometimes we want to retrieve multiple values in a single transaction and already have a struct containing all those values as members. For example, we have 
```
struct CommonUsedStruct {
   T1 a1, a2;
   T2 b1;
   T3 c1;
} obj;
```
and in a transaction, we want to read `a1, a2, c1` and assign them to `obj.a1, .a2, c1`. Shall we always create another struct that may be used just once
```
struct RarelyUsedStruct {
  T1 a1, a2;
  T3 c1;
}
```
for this return value? I think this scenario is common in DD when I look at the code.
I think I can comment this function out for now, and probably add some extra notes in the comment. 
Any thoughts here, guys?

Reply:
I don't have a strong preference on this, just curious how it will be used. I am fine keeping it as.